main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
+//  Noun:       screwdriver
+//  action 1:   the screwdriver screws 
+screwdriver.screw();
+//  action 2:   the screwdriver unscrews
+screwdriver.unscrew();
+//  action 3:   the screwdriver strips
+screwdriver.strip();
 
-//  action 2:
-
-//  action 3:
- 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:       torch
+//  action 1:   the torch burns
+torch.burn();
+//  action 2:   the torch lights
+torch.light();
+//  action 3:   the torch shows the direction of wind
+torch.showDirectionOfWind();
 
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog eats
+dog.eat();
+//  action 2:   the dog fetches
+dog.fetch();
+//  action 3:   the dog finds you
+dog.findsYou();
  
 //  4)
-//  Noun:
-//  action 1:
+//  Noun:       synth
+//  action 1:   the synth sequences
+synth.sequence();
+//  action 2:   the synth arpeggiates
+synth.arp();
+//  action 3:   the synth modulates
+synth.modulate();
 
-//  action 2:
-
-//  action 3:
- 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun:       tapemachine
+//  action 1:   the tapemachine playacks
+tapemachine.playback();
+//  action 2:   the tapemachine stops
+tapemachine.stop();
+//  action 3:   the tapemachine reverses
+tapemachine.reverse();
```

Comment:
`tapemachine` is 2 words in english, so you should use `camelCaseToSpellIt`
- [ ] revise